src/CreateEventQueryView/CreateEventQueryView.js

```diff
@@ -26,6 +26,7 @@ class CreateEventQueryView extends ConnectionComponent {
 			descriptionErrorMessage: ''
 		};
 		this.isCreateView = typeof this.props.match.params.eventQueryId === 'undefined';
+		this.newestNotificationShowns = false;
```

Comment:
Please remove the typo from the variable name and maybe latestNotificationShown would be better